Doc/howto/descriptor.rst

```diff
@@ -1477,20 +1501,30 @@ Python:
     class Object:
         'Simulate how object.__new__() allocates memory for __slots__'
 
-        def __new__(cls, *args):
+        def __new__(cls, *args, **kwargs):
```

Comment:
I'll review the rest of these later.   The title "fix the Python equivalents" implies that these we broken examples.    Many of these proposed edits are stylistic and exceed what the document was attempting to accomplish. 

The code existing is tested for fidelity to the C version and covers the essential points that we're trying to communicate.  There is no goal to make a drop-in replacement or to add code that distracts from the essentials.  User/reader testing shows that the existing code is already near the outer limits of what readers can absorb.

Reply:
I agree, I have updated the title.